plugins/org.eclipse.osee.framework.skynet.core/src/org/eclipse/osee/framework/skynet/core/transaction/TransactionManager.java

```diff
@@ -471,4 +475,19 @@ public static void persistInTransaction(String comment, Artifact... artifacts) {
       transaction.execute();
    }
 
+   public static TransactionRecord getTransactionRecord(TransactionId txId) {
```

Comment:
Provided the RelationLoader comments prove to be true and you don't end up needing this there...Do we need this? It would only be used in a test at that point. + it's always good to get rid of caches :)

Reply:
TransactionRecords don't ever change, so this cache isn't bad.  Agreed on getting rid of caches that have to be kept up to date or cleared.